ts-defs/index.d.ts

```diff
@@ -756,6 +756,23 @@ interface TypeActionOptions extends ClickActionOptions {
     paste?: boolean;
 }
 
+interface DragToElementOptions extends MouseActionOptions {
+    /**
+     * Mouse pointer X coordinate that define a point where the dragToElement action is finished.
+     * If an offset is a positive integer, coordinates are calculated relative to the top-left corner of the destination element.
+     * If an offset is a negative integer, they are calculated relative to the bottom-right corner.
+     * The default is the center of the destination element.
+     */
+    destinationOffsetX?: number;
+    /**
+     * Mouse pointer Y coordinate that define a point where the dragToElement is finished started.
+     * If an offset is a positive integer, coordinates are calculated relative to the top-left corner of the destination element.
+     * If an offset is a negative integer, they are calculated relative to the bottom-right corner.
+     * The default is the center of the destination element.
```

Comment:
By default, the dragToElement action is finished in the center of the destination element.